addons/l10n_ar_base/models/res_partner_bank.py

```diff
@@ -0,0 +1,51 @@
+##############################################################################
+# For copyright and license notices, see __manifest__.py file in module root
+# directory
+##############################################################################
+from odoo import fields, models, api, _
+from odoo.tools import pycompat
+from odoo.exceptions import ValidationError
+
+
+class ResPartnerBank(models.Model):
+
+    _inherit = 'res.partner.bank'
+
+    l10n_ar_cbu = fields.Char(
+        'CBU',
+        help="Argentine Banking Unique Code",
+    )
```

Comment:
As you don't use the IBAN, can not we use the bank account number itself and make a type that is CBU?

Reply:
To identify an account on Argentina, you can have:
* CBU: a long number unique for all argentinian accounts
* account number: a number only unique for that bank, used when you transfer between the same bank
So that's why we add a new field and use both.

As this field is actually not used by other modules and there is a work around where you can create two banks account one with the CBU and another with the "account number" we think it's better to remove this from the PR

Do you agree?